### PR TITLE
removed participant-gitops namespace

### PR DESCRIPTION
**Pull request by SkylarScaling**

### CODE REVIEW

File: ansible/participants-setup.yaml [hunk 1]
```diff
@@ -1,36 +1,7 @@
 ---
 - name: Setup Participant
   hosts: local
-  vars:
-    participant_namespace: "{{ participant }}-gitops"
   tasks:
-    - name: "{{ participant_namespace }} namespace is {{ state }}"
```
Comment (darthlukan) [inline ansible/participants-setup.yaml:7]: Looks good to me!

**Review by darthlukan — APPROVED** [review]
Looks good to me, the linter shouldn't complain :)

File: ansible/participants-setup.yaml (hunk 1 above)
Comment (SkylarScaling) [reply ansible/participants-setup.yaml:7]: The linter didn't complain, it just died trying to set itself up 🙃

**Comment by darthlukan** [comment]
Merging anyways. The linter task needs to be updated apparently, but this playbook is correct.

**Pull request merged**